stytch/b2b/organizations/members/types.go

```diff
@@ -0,0 +1,233 @@
+	// are server errors.
+	StatusCode int32 `json:"status_code,omitempty"`
+}
+
+// DeleteResponse: Response type for `Members.Delete`.
+type DeleteResponse struct {
+	// RequestID: Globally unique UUID that is returned with every API call. This value is important to log for
+	// debugging purposes; we may ask for this value to help identify a specific API call when helping you
+	// debug an issue.
+	RequestID string `json:"request_id,omitempty"`
+	// MemberID: Globally unique UUID that identifies a specific Member.
+	MemberID string `json:"member_id,omitempty"`
+	// StatusCode: The HTTP status code of the response. Stytch follows standard HTTP response status code
+	// patterns, e.g. 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX
+	// are server errors.
+	StatusCode int32 `json:"status_code,omitempty"`
+}
+
+// GetResponse: Response type for `Members.Get`.
+type GetResponse struct {
+	// RequestID: Globally unique UUID that is returned with every API call. This value is important to log for
+	// debugging purposes; we may ask for this value to help identify a specific API call when helping you
+	// debug an issue.
+	RequestID string `json:"request_id,omitempty"`
+	// MemberID: Globally unique UUID that identifies a specific Member.
+	MemberID string `json:"member_id,omitempty"`
+	// Member: The [Member object](https://stytch.com/docs/b2b/api/member-object).
+	Member organizations.Member `json:"member,omitempty"`
+	// Organization: The [Organization object](https://stytch.com/docs/b2b/api/organization-object).
+	Organization organizations.Organization `json:"organization,omitempty"`
+	// StatusCode: The HTTP status code of the response. Stytch follows standard HTTP response status code
+	// patterns, e.g. 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX
+	// are server errors.
+	StatusCode int32 `json:"status_code,omitempty"`
+}
+
+// SearchResponse: Response type for `Members.Search`.
+type SearchResponse struct {
+	// RequestID: Globally unique UUID that is returned with every API call. This value is important to log for
+	// debugging purposes; we may ask for this value to help identify a specific API call when helping you
+	// debug an issue.
+	RequestID string `json:"request_id,omitempty"`
+	// Members: An array of [Member objects](member-object).
+	Members []organizations.Member `json:"members,omitempty"`
+	// ResultsMetadata: The search `results_metadata` object contains metadata relevant to your specific query
+	// like `total` and `next_cursor`.
+	ResultsMetadata organizations.ResultsMetadata `json:"results_metadata,omitempty"`
+	// Organizations: A map from `organization_id` to [Organization
+	// object](https://stytch.com/docs/b2b/api/organization-object). The map only contains the Organizations
+	// that the Members belongs to.
+	Organizations []OrganizationsEntry `json:"organizations,omitempty"`
```

Comment:
This doesn't look quite right to me - shouldn't this be a `map[string]Organization`?